roles/copy_cert_files/tasks/main.yml

```diff
@@ -45,3 +45,23 @@
     dest: "{{ item.dest }}"
     state: link
   loop: "{{ migrid_cert_file_links }}"
+
+# copy selfsigned root certificate to external-certificates/
+
+- name: Create directory for external-certificates/ if needed
+  file:
+    path: "{{ migrid_root }}/external-certificates"
+    owner: "{{ migrid_adm }}"
+    group: "{{ migrid_adm }}"
+    state: directory
+    mode: "0755"
+  when: migrid_docker_compose_override==true
+
+- name: Copy selfsigned certificate if needed
+  copy:
+    src: "{{migrid_copy_cert_files[0].src}}{{migrid_selfsigned_cert_file}}"
```

Comment:
making concatenated paths like this breaks the configurability of migrid_copy_cert_files. One cannot have that empty for example.
Instead please use just `migrid_selfsigned_cert_file` here and set a default value which can contain arbitrary concatenations.